sonar-project.properties

```diff
@@ -0,0 +1,20 @@
+sonar.projectKey=ita-social-projects_SpaceToStudy-ClientMobile
+sonar.organization=ita-social-projects
+
+sonar.projectName=SpaceToStudy-ClientMobile
+
+# Path is relative to the sonar-project.properties file.
+sonar.sources=.
+sonar.exclusions=**/src/tests/**/*, **/jest.config.js
```

Comment:
You can also add all the files related to the styles in the exclusions

Reply:
Thank you for your suggestion! However, my intention was not to remove the style files from analysis entirely. I included them in sonar.coverage.exclusions to ensure they are excluded only from coverage calculations, not from other analyses. This way, the style files will still be analyzed, but they won't impact the test coverage results.